label_studio/tasks/mixins.py

```diff
@@ -14,6 +14,25 @@ def update_is_labeled(self, *args, **kwargs) -> None:
     def post_process_bulk_update_stats(cls, tasks) -> None:
         pass
 
+    def after_delete_actions(self):
```

Comment:
Why do we need these 3 functions if they are not doing anything?

Reply:
It won't work in LSO if there are no such methods in LSO mixin. 
I will clean up unused one.